scripts/start.js

```diff
@@ -1,122 +0,0 @@
-/* eslint-disable no-console */
```

Comment:
Thank you for the catch - this was never intended to be deleted - it looks like it was accidentally deleted instead of the testing util script file - it's now added back.